yt_dlp/extractor/gronkh.py

```diff
@@ -56,6 +63,8 @@ def _real_extract(self, url):
             'upload_date': unified_strdate(data_json.get('created_at')),
             'formats': formats,
             'subtitles': subtitles,
+            'duration': data_json.get('source_length'),
+            'chapters': chapters,
```

Comment:
```suggestion
            'duration': float_or_none(data_json.get('source_length')),
            'chapters': traverse_obj(data_json, (
                'chapters', lambda _, v: float_or_none(v['offset']) is not None, {
                    'title': 'title',
                    'start_time': ('offset', {float_or_none}),
                })) or None,
```

import `float_or_none`